amundsen_application/static/js/components/TableDetail/TableIssues/styles.scss

```diff
@@ -27,12 +23,48 @@
     white-space: nowrap;
     overflow: hidden;
     text-overflow: ellipsis;
+    max-width: fit-content;
   }
   .issue-title-display-text {
     display: flex;
     max-width: fit-content;
   }
-  .table-issue-more-issues {
-    margin: 0px 2px 0px 2px; 
+  .table-issue-status {
+    display: flex;
+    justify-content: flex-end;
+    flex-grow: 1.1;
+    min-width: fit-content;
+    margin: 0px 0px 0px 5px;
```

Comment:
Can we use `$spacer-1` instead of 5px? 

Not sure if this was an oversight in Figma, but when we were using Abstract our designers had given us component breakdowns and one of the concepts was that the default spacer size was "8px (the value of  $spacer-1)" and that all other spacing would be multiples of 8. So I'd question is this was actually supposed to be 5.